src/RxContainer.js

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 
+const getDevToolsExt = () => {
+  if (process.env.NODE_ENV === 'development' && typeof window !== 'undefined') {
+    return window.devToolsExtension || window.__REDUX_DEVTOOLS_EXTENSION__;
```

Comment:
The 2nd variable is the one that's expected in the newer versions of the tool. Should we have change the order of them?